api/src/main/resources/workbench.yaml

```diff
@@ -1207,6 +1224,40 @@ definitions:
   RegistrationRequest:
     type: object
 
+  IdVerificationRequest:
+    type: object
+    properties:
+      firstName:
+        type: string
+      lastName:
+        type: string
+      streetLine1:
+        type: string
+      streetLine2:
+        type: string
+      city:
+        type: string
+      state:
+        type: string
+      zip:
+        type: string
+      dob:
+        type: string
+      documentType:
```

Comment:
What are documentType and documentNumber? (Is this like "driver's license" and "driver's license ID"?) Add description fields to indicate.

Reply:
Added documentation reference.